translations/en.json

```diff
@@ -790,13 +790,15 @@
     "not-keycard-text": "The card you used is not a Keycard. You need to purchase a Keycard to use it",
     "not-keycard-title": "Not a Keycard",
     "notifications": "Notifications",
+    "show-notifications": "Show notifications",
     "notifications-settings": "Notifications",
-    "notification-settings": "Notification settings",
+    "notification-settings": "Notifications",
```

Comment:
do we need two keys with the same copy ?

Reply:
good point :)